README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+We use the following icons for our organization and repository logos:
+- [Gear icon](https://thenounproject.com/search/?q=gear&i=102547) by Jardson Almeida, from the Noun Project
```

Comment:
Sorry, no style guide =/ It's mostly personal style developed over the years to avoid Markdown pitfalls but save on space.

The reason for the extra line between paragraph/lists is they are different elements and sometimes formatters place them on the same line (e.g. restructuredText).

The reason for the no extra line between heading and first line is syntax highlighting typically bolds the heading so it has enough visual weight and they are typically attached thoughts.